code/socialDistribution/requests.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import json
+
+
+def get(url, params=None):
```

Comment:
Just a minor thing
I think there was a make_request function in the utility.py file. Let's replace it with the ones you wrote here. If you'll make the change, I think the like_post view in the socialDistribution/views.py uses the make_request.

Reply:
Ya, that sounds good. That was my end goal as well (to clean things up), just wasn't sure what else relies on make_request. If that's the only place it is used (and no other active PRs rely on make_request) I can delete/replace it now. 

This new file was essentially the same thing but putting all those functions in a more cohesive file.

Reply:
Actually a few of the active PRs use make_request so I think it would be best handled in a quick PR later tonight after everything else is merged. What do you think?

Reply:
Yup, that sounds good to me 👍

Reply:
I think it's my pr for #161 that uses the make_request
I can change it after we merge our pr's into master